Makefile

```diff
@@ -0,0 +1,5 @@
+lint:
+	flake8 place tests
+
+test:
```

Comment:
What's this Makefile doing?

Reply:
Just so I can do `make test` and `make lint`. Will be expanded to do more things as more things are built (push to PyPi, etc).

Reply:
Ah ok cool.